tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc

```diff
@@ -473,5 +473,4 @@ GlobalTracer.registerIfAbsent(tracer);
 
 [[sec:microprofile-telemetry-and-microprofile-opentracing]]
 == MicroProfile Telemetry and MicroProfile OpenTracing
-If MicroProfile Telemetry and MicroProfile OpenTracing are both present in one application, it is advised only to enable one of them.
-Otherwise, no portable behaviour may occur.
+If MicroProfile Telemetry and MicroProfile OpenTracing are both present in one application, it is advised only to enable one of them. Otherwise, no portable behaviour may occur.
```

Comment:
```suggestion
If MicroProfile Telemetry and MicroProfile OpenTracing are both present in one application, it is recommended to only enable one of them, otherwise non-portable behaviour may occur.
```